kalite/main/api_resources.py

```diff
@@ -71,110 +71,6 @@ class Meta:
         authorization = UserObjectsOnlyAuthorization()
 
 
-class Video:
```

Comment:
Wow, cool. Where does video data get loaded from now...? (Probably going to see it below)